### DISCUSSION TITLE
Crashing on Linux Mint when trying to import audio file

### DISCUSSION THREAD

**Discussion by RaymondTheBrave0** [chidiwilliams/buzz#600] [General]
I down loaded the buzz-0.8.3.deb file and installed it successfully. However when I try and run it Buzz comes up and shows the front screen. If I try and import an audio file it just disappears which I assume it has crashed.
![Screenshot from 2023-09-16 22-29-13](https://github.com/chidiwilliams/buzz/assets/78306833/2659bb11-b058-49ca-a1d1-1e83a19701b6)
 I am attaching the screen as I see it.   

I am running Linux Mint 21 using Cinnamon                   
Kernel: Linux 5.15.0-83-generic
Architecture: x86-64
Hardware Vendor: Gigabyte Technology Co., Ltd.
 Hardware Model: X570 AORUS ULTRA
32GB Ram

I have used Buzz on my Windows 11 which runs on the same hardware without any problems. 

What more info do you need?
Is there a log somewhere?

Thanks Raymond

**Comment by faveoled** [comment]
You should try running it from the Terminal
